### ISSUE TITLE
cr2 format causes trouble during fork

### ISSUE THREAD

**Issue by lbestock** [urapadmin/kiosk#1002]
the file is 14e67f7d-84e3-4f8a-bae3-846728c2ffb8 in the pvd file repository
it was included in the fork upstream because of a file picking rule that addressed it as a single file

a) Why is it not possible to create a thumbnail? There are thumbnails for it in the file repository
b) why did fork stop? Do we really want that? I thought there was a setting in the configuration that ignored such errors.

Here is the log:

[MCP_10140_urap_Fri_020421-1851.log](https://github.com/urapadmin/kiosk/files/6251529/MCP_10140_urap_Fri_020421-1851.log)

**Comment by urapadmin** [comment]
I cannot reproduce this behavior. I tried it with this file: fork works. I deleted it and uploaded it again as a new file. And then fork worked again. 
Why in your case the "dest_format could not be determined" is beyond me. But since I cannot reproduce it, I also cannot debug it.
We have to wait until something like this happens again and then I have to look into it immediately. Between April and now too many versions and synchronizations have blurred the picture.
